src/components/img-title-description/ImgTitleDescription.js

```diff
@@ -0,0 +1,32 @@
+import { Box } from '@mui/material'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+
+const style = {
```

Comment:
Change this name to "componentStyle". And then use it in a component like { style = componentStyle  }.

Reply:
ok